lua/telescope/builtin.lua

```diff
@@ -112,6 +112,13 @@ end
 
 builtin.quickfix = function(opts)
```

Comment:
Can you add an opt for `loclist_fallback`?

Reply:
Or.... hmm, maybe what we want is just two different functions.

Reply:
They mean very different things.

Reply:
Done!